src/System.Management.Automation/engine/InitialSessionState.cs

```diff
@@ -5962,9 +5962,11 @@ internal static class PSSnapInHelpers
                 {"Connect-PSSession",                 new SessionStateCmdletEntry("Connect-PSSession", typeof(ConnectPSSessionCommand), helpFile) },
                 {"Debug-Job",                         new SessionStateCmdletEntry("Debug-Job", typeof(DebugJobCommand), helpFile) },
                 {"Disable-PSSessionConfiguration",    new SessionStateCmdletEntry("Disable-PSSessionConfiguration", typeof(DisablePSSessionConfigurationCommand), helpFile) },
-                {"Disable-PSRemoting",                new SessionStateCmdletEntry("Disable-PSRemoting", typeof(DisablePSRemotingCommand), helpFile) },
                 {"Disconnect-PSSession",              new SessionStateCmdletEntry("Disconnect-PSSession", typeof(DisconnectPSSessionCommand), helpFile) },
+#if !UNIX
+                {"Disable-PSRemoting",                new SessionStateCmdletEntry("Disable-PSRemoting", typeof(DisablePSRemotingCommand), helpFile) },
```

Comment:
We should go ahead and disallow all PSSession commands that we know are not supported in Linux (WinRM specific).  I believe these are:
```powershell
Connect-PSSession
Disable-PSSessionConfiguration
Disconnect-PSSession
Enable-PSSessionConfiguration
Get-PSSessionCapability
Get-PSSessionConfiguration
New-PSSessionConfigurationFile
New-PSSessionOption
Receive-PSSession
Register-PSSessionConfiguration
Set-PSSessionConfiguration
Test-PSSessionConfigurationFile
Unregister-PSSessionConfiguration
```